tests/StarkEx.Client.SDK.Tests/Clients/Spot/SpotGatewayClientTests.cs

```diff
@@ -461,62 +461,62 @@ public async Task AddTransactionAsync_MintTokenIdIsInvalid_PostRequestExceptionI
+        yield return new object[] { "{\"code\": \"StarkErrorCode.MISSING_BLOCKCHAIN_ID\"}", SpotApiCodes.MissingBlockchainId };
+        yield return new object[] { "{\"code\": \"StarkErrorCode.MISSING_FEE_OBJECT\"}", SpotApiCodes.MissingFeeObject };
+        yield return new object[] { "{\"code\": \"StarkErrorCode.NESTED_MULTI_TRANSACTION\"}", SpotApiCodes.NestedMultiTransaction };
+        yield return new object[] { "{\"code\": \"StarkErrorCode.ORDER_OVERDUE\"}", SpotApiCodes.OrderOverdue };
+        yield return new object[] { "{\"code\": \"StarkErrorCode.OUT_OF_RANGE_AMOUNT\"}", SpotApiCodes.OutOfRangeAmount };
+        yield return new object[] { "{\"code\": \"StarkErrorCode.OUT_OF_RANGE_BALANCE\"}", SpotApiCodes.OutOfRangeBalance };
+        yield return new object[] { "{\"code\": \"StarkErrorCode.OUT_OF_RANGE_BATCH_ID\"}", SpotApiCodes.OutOfRangeBatchId };
+        yield return new object[] { "{\"code\": \"StarkErrorCode.OUT_OF_RANGE_ETH_ADDRESS\"}", SpotApiCodes.OutOfRangeEthAddress };
+        yield return new object[] { "{\"code\": \"StarkErrorCode.OUT_OF_RANGE_EXPIRATION_TIMESTAMP\"}", SpotApiCodes.OutOfRangeExpirationTimestamp };
+        yield return new object[] { "{\"code\": \"StarkErrorCode.OUT_OF_RANGE_FIELD_ELEMENT\"}", SpotApiCodes.OutOfRangeFieldElement };
+        yield return new object[] { "{\"code\": \"StarkErrorCode.OUT_OF_RANGE_NONCE\"}", SpotApiCodes.OutOfRangeNonce };
+        yield return new object[] { "{\"code\": \"StarkErrorCode.OUT_OF_RANGE_ORACLE_PRICE_QUORUM\"}", SpotApiCodes.OutOfRangeOraclePriceQuorum };
+        yield return new object[] { "{\"code\": \"StarkErrorCode.OUT_OF_RANGE_ORDER_ID\"}", SpotApiCodes.OutOfRangeOrderId };
+        yield return new object[] { "{\"code\": \"StarkErrorCode.OUT_OF_RANGE_POSITIVE_AMOUNT\"}", SpotApiCodes.OutOfRangePositiveAmount };
+        yield return new object[] { "{\"code\": \"StarkErrorCode.OUT_OF_RANGE_PUBLIC_KEY\"}", SpotApiCodes.OutOfRangePublicKey };
+        yield return new object[] { "{\"code\": \"StarkErrorCode.OUT_OF_RANGE_SIGNATURE_SUBFIELD\"}", SpotApiCodes.OutOfRangeSignatureSubfield };
+        yield return new object[] { "{\"code\": \"StarkErrorCode.OUT_OF_RANGE_TOKEN_ID\"}", SpotApiCodes.OutOfRangeTokenId };
+        yield return new object[] { "{\"code\": \"StarkErrorCode.OUT_OF_RANGE_VAULT_ID\"}", SpotApiCodes.OutOfRangeVaultId };
+        yield return new object[] { "{\"code\": \"StarkErrorCode.REPLACED_BEFORE\"}", SpotApiCodes.ReplacedBefore };
+        yield return new object[] { "{\"code\": \"StarkErrorCode.REQUEST_FAILED\"}", SpotApiCodes.RequestFailed };
+        yield return new object[] { "{\"code\": \"StarkErrorCode.SCHEMA_VALIDATION_ERROR\"}", SpotApiCodes.SchemaValidationError };
+        yield return new object[] { "{\"code\": \"StarkErrorCode.TRANSACTION_CANCELLED\"}", SpotApiCodes.TransactionCancelled };
         yield return new object[] { "{\"code\": \"TRANSACTION_PENDING\"}", SpotApiCodes.TransactionPending };
```

Comment:
You forgot to add it here?

Reply:
This is the only successful code, hence why it doesn't have the prefix. Tested on the playground